collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -144,6 +170,16 @@ public boolean onError(MediaPlayer mp, int what, int extra) {
         addHelpTextLayout(getHelpTextLayout());
     }
 
+    //source::https://stackoverflow.com/questions/18996183/identifying-rtl-language-in-android/23203698#23203698
```

Comment:
How about: `// Source: https...`
The `source::https://` looks like some new kind of “meta protocol”. Maybe it is?